docs/onboarding_to_argocd.md

```diff
@@ -0,0 +1,123 @@
+# ArgoCD onboarding procedure
+
+We have an ArgoCD instance deployed on MOC that can be used to deploy the application manifests located in this repo.
+
+The ArgoCD instance is deployed on [MOC CNV Sandbox](https://github.com/open-infrastructure-labs/moc-cnv-sandbox) cluster.
```

Comment:
Isn't this subject to change in few days anyway? I'd remove the link and rather point to our continuous-delivery repo, where the actual codebase lives.

Reply:
updated